SU2_CFD/src/output_structure.cpp

```diff
@@ -3998,6 +3998,11 @@ void COutput::SetRestart(CConfig *config, CGeometry *geometry, CSolver **solver,
   bool dual_time = ((config->GetUnsteady_Simulation() == DT_STEPPING_1ST) ||
                     (config->GetUnsteady_Simulation() == DT_STEPPING_2ND));
 
+  int rank = MASTER_NODE;
```

Comment:
same here

Reply:
You can also remove the declaration of the 'rank' variable as it is now also a member variable of most of the classes (including COutput).